README.md

```diff
@@ -51,41 +51,52 @@ coverage html
 
 ### Your own coverage tool
 
-<The following is supposed to be repeated for each group member>
+#### 1. Levente Szabó
 
-<Group member name>
+* Function 1: `versions_from_parent_dir`
 
-<Function 1 name>
+* Function 2: `get_versions` 
 
-<Show a patch (diff) or a link to a commit made in your forked repository that shows the instrumented code to gather coverage measurements>
+Commit [3f36a45](https://github.com/Papajo-boban/shapely/commit/3f36a4572a07c086d23f273623df349281e9176d) creates the CoverageTracker class, and commit [a7b54e6](https://github.com/Papajo-boban/shapely/commit/a7b54e625f1406bc9cb63f64b2b8b080ace2444c) instruments both functions. The coverage tool can be executed by running the `cov_info.sh` script located in the `shapely/cov_tracker_levi/` directory.
 
-<Provide a screenshot of the coverage results output by the instrumentation>
+Coverage results:
 
-<Function 2 name>
-
-<Provide the same kind of information provided for Function 1>
+<img width="426" alt="Screenshot 2024-06-08 at 17 32 05" src="https://github.com/Papajo-boban/shapely/assets/134519958/b9a489d6-374a-4526-959a-1bf6bcd0d99a">
 
 ## Coverage improvement
 
 ### Individual tests
 
-<The following is supposed to be repeated for each group member>
+#### 1. Levente Szabó
+
+>***Note:*** *I misunderstood and thought we had to write tests that cover all branches of the 2 chosen functions, that's why I created more than 2 tests.*
```

Comment:
The requirements have changed so it is not necessary.
Now you need for a method to cover more than 80% so you did it well

Reply:
Done!